### DISCUSSION TITLE
The coordinator is pending

### DISCUSSION THREAD

**Discussion by dby2017** [alibaba/GraphScope#301] [Q&A] [answered]
Hello，
when I  started k8s, I encounted this problem "defalult pod/graphscope030i1-conditioner 0/1 Pending 0 g 65s.  "  How to solve the problem?

**Comment by sighingnow** [comment]
Hi @dby2017,

I have moved this ticket to the "discussion" tab here to make the discussion could help other users who meet the same problems.

I'm also ccing @lidongze0629 and @siyuan0322 who could provide more related information.

Best regards,
Tao

**Reply by dby2017** [reply]
Thank you!

**Comment by lidongze0629** [comment, marked as answer]
Hi, @dby2017 
Could you provide more details to describe this problem ? Such as full log，the version of graphscope .

**Describe the bug**
A clear and concise description of what the bug is.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**
 - GraphScope version: [e.g., v0.1, v0.2.2, master]
 - OS: [e.g. MacOS, Linux]
 - Version [e.g. 10.15]
 - Kubernetes Version [e.g., 1.19]

**Additional context**
Add any other context about the problem here.

**Reply by dby2017** [reply]
hi @lidongze0629 ，
Below is the information of my environment：
- GraphScope version   :   graphscope-0.3.0
- OS   :  CentOS Linux release 7.9.2009 (Core)  ,  vmware envirments on  Laptop  machine  
- Kubernetes Version  :   kubernetes  1.21.0
![image](https://user-images.githubusercontent.com/32199053/118072032-97262000-b3db-11eb-9cce-eda325896b66.png)

Thank you!

**Reply by lidongze0629** [reply]
@dby2017 

From your screenshots , I believe you have two separate problems.

**Problem 1 - CrashLoopBackOff of the pod**
you can use `kubectl -n <ns> describe pod <pod>` and `kubectl -n <ns> logs -f <pod>` command to view the error message .

**Problem 2 - Pending**

I found multiple GraphScope instances in your k8s cluster.  Maybe the cpu(4 cores) or memory (5G) is not enough for launching multiple instances. you can clear and relaunch it.

**Comment by sighingnow** [comment]
May I know the environement settings in your case? e.g.,

+ Are you working with a "kind" cluster on a laptop/desktop machine?
+ How many cores are available?
+ And the available physical memory size?

**Reply by dby2017** [reply]
hi @sighingnow ,
The environment settings in my case:
- OS   :  CentOS Linux release 7.9.2009 (Core)  ,  vmware envirments on  Laptop  machine  
- Cores 4 on per machine
- available physical memory size :  5G on per machine
